go-controller/pkg/ovn/controller/apbroute/network_client.go

```diff
@@ -299,24 +299,24 @@ func (nb *northBoundClient) addGWRoutesForPod(gateways []*gateway_info.GatewayIn
 
 // AddHybridRoutePolicyForPod handles adding a higher priority allow policy to allow traffic to be routed normally
 // by ecmp routes
-func (nb *northBoundClient) addHybridRoutePolicyForPod(podIP net.IP, node string) error {
+func (nb *northBoundClient) addHybridRoutePolicyForPod(podIP string, node string) error {
```

Comment:
In instances like this, wouldn't it make sense to keep the type as `net.IP` and just convert to slice when using the address set methods, so that we could keep `utilnet.IsIPv6` which is probably more performant?

Reply:
done.